readme.md

```diff
@@ -1,7 +1,7 @@
 # Frontend practice with catalog page
 Replace `<your_account>` with your Github username and copy the links to Pull Request description:
-- [DEMO LINK](https://<your_account>.github.io/layout_catalog/)
-- [TEST REPORT LINK](https://<your_account>.github.io/layout_catalog/report/html_report/)
+- [DEMO LINK](https://<BiletskyiVolodymyr>.github.io/layout_catalog/)
+- [TEST REPORT LINK](https://<BiletskyiVolodymyr>.github.io/layout_catalog/report/html_report/)
```

Comment:
Fix your links. And there is smth wrong with your deployment
<img width="1680" alt="image" src="https://user-images.githubusercontent.com/99403198/213684653-fa1821a5-1442-47e1-bfeb-5ac67ba17acb.png">

```suggestion
- [DEMO LINK](https://BiletskyiVolodymyr.github.io/layout_catalog/)
- [TEST REPORT LINK](https://BiletskyiVolodymyr.github.io/layout_catalog/report/html_report/)
```